fast-af/fast-checkout-woocommerce
# PINT-387 Add PDP hook option

# Description

Add an option to select the hook used to display the PDP button. Currently, the hook used to display the PDP button is hard-coded into the plugin with no option to choose a different hook/location for the button. This update provides 4 options: Before Quantity Selection, After Quantity Selection, After Add to Cart Button, and Other.

If the "Other" option is selected, the admin can enter a custom hook to be used. This option is for advanced use only, but it provides a possible way to overcome limitations with some sellers that use their own custom templates.

# Testing instructions

1. Login to the [dev site admin](https://fasttestdev.wpcomstaging.com/wp-admin).
2. Go to the [Fast settings page](https://fasttestdev.wpcomstaging.com/wp-admin/admin.php?page=fast).
3. Scroll down to the "Select Product Button Location" setting.
4. Select a button location.
5. View a product details page and verify that the button is in the location you selected.

# Checks

@ilkerulutas @brikr this is ready for review.
